lib/scrabble.rb

```diff
@@ -1,9 +1,18 @@
 class Scrabble
+  attr_accessor :word
+
+  def initialize
+    @word = word
```

Comment:
Not really using this. Be sure that it's clear in your mind when you need an initialize method, how to set instance variables, and when we need to.

Everything in initialize runs when we call `.new` on the Scrabble class. Currently all that's happening in this method is that we're setting the instance variable `@word` equal to `nil` (since `word` isn't given a value anywhere else it evaluates to `nil`).

Since we're passing a word into our `#score` method, we don't really need to have `@word` as an instance variable.